generators/app/templates/src/StarterKit/Enrichers/TypeEnricher.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Data;
+using System.Data.Common;
+using Microsoft.EntityFrameworkCore;
+using Serilog.Core;
+using Serilog.Events;
+
+namespace StarterKit.Enrichers
+{
+    public class TypeEnricher : ILogEventEnricher
+    {
+        public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
+        {
+            var type = string.Empty;
+
+            switch (logEvent.Exception)
```

Comment:
Ik heb het tijdens je demo niet gezien, maar is het bij exceptions duidelijk dat dit om een fout gaat? Kan je die logs makkelijk onderscheiden van gewone logs? (Staat los van deze enricher, maar ik vraag het me af.)

Reply:
Indien er een exception bij zit dan wordt er ook een extra property in de JSON output (exception) weggeschreven. Je zou dus makkelijk kunnen filteren op die property om te zien of het om een exception gaat.